common/build/eslint-plugin-fluid/.prettierignore

```diff
@@ -0,0 +1,11 @@
+# Generated by npm / Lerna
```

Comment:
I don't think these are right. This package doesn't have a `package-lock.json`, or printed-configs or type-tests. It might be better to remove this file and specify the `.prettierignore` at the repo root instead in the prettier package.json scripts.

Reply:
You're referring to this `.prettierignore` file located in the root of the client release group right? 

https://github.com/microsoft/FluidFramework/blob/32cfa0952b490e0443d56aa86780f92d852d4aae/.prettierignore#L1

`eslint-plugin-fluid` generates `nyc` and `node_modules` and these are already covered in the file, so I believe we wouldn't need any more addition.